tests/test1009.py

```diff
@@ -0,0 +1,45 @@
+# test some ligatures and fractions close to U0000. Avoid high values such as
+# UFxxx since the Unicode chart may add new unicode values later.
+
+import sys, fontforge
+
+print "Get Table Array Totals."
```

Comment:
Ok yep. The reason why it wasn't failing on Travis when it should have (which compiles FontForge with Python3 scripting) is because `test1009.py` wasn't in `Makefile.am`.